packages/sources/src/components/justify-form-groups/justify-form-groups.models.ts

```diff
@@ -0,0 +1,2 @@
+export interface JustifyFormGroups {
```

Comment:
In de body van het issue heb ik een modelering voorgesteld, die graag inzetten en op doorbouwen